seL4-doc/src/appendices/tftpboot_example.md

```diff
@@ -0,0 +1,53 @@
+# Setting up a netboot server using TFTP - Instructions for Linux
+
+The most efficient way of booting from an image is to set up a TFTP server from which the board can read from. This removes the need to repeatedly flash an SD card with a new image, and instead the image can just be copied to the appropriate directory where it can be read and booted. The strategy used will depend on the operating system used, but this guide is for Linux.
+
+1. [Dnsmasq](https://thekelleys.org.uk/dnsmasq/doc.html) provides a network infrastructure for small networks and is designed to be lightweight. The program can be installed using apt-get
+
+```
+sudo apt-get install dnsmasq
+```
+
+2. Create the root folder where the image will be stored. In this example, the path will be `/var/lib/tftpboot`.
+
+2. Identify the ethernet interface you will be connecting the board to using `ip addr` command from the terminal.
+
+![Command `ip addr` output](../figures/ip-addr-ethernet.png)
+
+From this command, the necessary information is the *mac address* of the interface (01:23:45:67:89:0a in this case), and the *ip address* (10.0.0.253).
+
+3. Edit the file `/etc/dnsmasq.conf` with the following content:
+
+```
+interface=eno2
+dhcp-range=10.0.0.101,10.0.0.200,12h
+dhcp-host=01:23:45:67:89:0a,svr1,infinite
+dhcp-boot=sel4_image
+enable-tftp
+tftp-root=/var/lib/tftpboot
+```
+
+where the values are replaced with those corresponding to the output of the `ip addr` command. The `dhcp-range` should act as a net mask to allow ip allocation to the board.
```

Comment:
Pedantically, dhcp-range is not a net mask, it's a range of IP addresses which dnsmasq will allocate to devices.